### ISSUE TITLE
Add "share" button in storage media selection menu

### ISSUE THREAD

**Issue by msalini** [mollyim/mollyim-android#91]
### Is there an existing request for this?

- [X] I have searched the existing issues

### Feature description

The current menu features 3 buttons (save/select all/delete). It is currently not possible to select multiple media files and share them in one action (first you need to save them and then share).

Would it be possible to cram an extra button in the menu or find an alternative menu solution that includes a share button? 

![Screenshot_Molly](https://user-images.githubusercontent.com/32898857/151572674-df86d3ad-ba90-46a4-83a8-49b439a7f7b1.jpg)

**Comment by Seb3thehacker** [comment]
Yes, but it probably isn't a priority right now. I would love to see this implemented though!
